runtime/gc_ctrl.c

```diff
@@ -308,7 +308,6 @@ CAMLprim value caml_gc_stat(value v)
 {
   value res;
   CAML_EV_BEGIN(EV_EXPLICIT_GC_STAT);
-  res = gc_full_major_exn();
   if (Is_exception_result(res)) goto out;
```

Comment:
`Is_exception_result` and `caml_raise_if_exception` need to be deleted along with the call to `gc_full_major_exn`.